From f87bbeac91e8b975910858acd27bc4b5f13f626f Mon Sep 17 00:00:00 2001
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Mon, 2 Jul 2018 15:31:29 +0100
Subject: [PATCH 1308/1670] ASoC: sof: skl, cnl, apl: cleanup - add comments
 and reorder funcs

Add comments describing funcs/code blocks and reorder funcs so features
are grouped.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
(cherry picked from commit cb84cdb5a892b05bc2db12fc57740a319a535115)
---
 sound/soc/sof/intel/apl.c | 8 ++++----
 sound/soc/sof/intel/cnl.c | 8 ++++----
 sound/soc/sof/intel/skl.c | 8 ++++----
 3 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/sound/soc/sof/intel/apl.c b/sound/soc/sof/intel/apl.c
index 9db5a24..96885d5 100644
--- a/sound/soc/sof/intel/apl.c
+++ b/sound/soc/sof/intel/apl.c
@@ -77,10 +77,10 @@ struct snd_sof_dsp_ops sof_apl_ops = {
 	.dbg_dump	= hda_dsp_dump,
 
 	/* stream callbacks */
-	.host_stream_open = hda_dsp_pcm_open,
-	.host_stream_close = hda_dsp_pcm_close,
-	.host_stream_hw_params = hda_dsp_pcm_hw_params,
-	.host_stream_trigger = hda_dsp_pcm_trigger,
+	.pcm_open	= hda_dsp_pcm_open,
+	.pcm_close	= hda_dsp_pcm_close,
+	.pcm_hw_params	= hda_dsp_pcm_hw_params,
+	.pcm_trigger	= hda_dsp_pcm_trigger,
 
 	/* firmware loading */
 	.load_firmware = hda_dsp_cl_load_fw,
diff --git a/sound/soc/sof/intel/cnl.c b/sound/soc/sof/intel/cnl.c
index 99c80d7..10c98eb 100644
--- a/sound/soc/sof/intel/cnl.c
+++ b/sound/soc/sof/intel/cnl.c
@@ -205,10 +205,10 @@ struct snd_sof_dsp_ops sof_cnl_ops = {
 	.dbg_dump	= hda_dsp_dump,
 
 	/* stream callbacks */
-	.host_stream_open = hda_dsp_pcm_open,
-	.host_stream_close = hda_dsp_pcm_close,
-	.host_stream_hw_params = hda_dsp_pcm_hw_params,
-	.host_stream_trigger = hda_dsp_pcm_trigger,
+	.pcm_open	= hda_dsp_pcm_open,
+	.pcm_close	= hda_dsp_pcm_close,
+	.pcm_hw_params	= hda_dsp_pcm_hw_params,
+	.pcm_trigger	= hda_dsp_pcm_trigger,
 
 	/* firmware loading */
 	.load_firmware = hda_dsp_cl_load_fw,
diff --git a/sound/soc/sof/intel/skl.c b/sound/soc/sof/intel/skl.c
index 86097bb..2738202 100644
--- a/sound/soc/sof/intel/skl.c
+++ b/sound/soc/sof/intel/skl.c
@@ -77,10 +77,10 @@ struct snd_sof_dsp_ops sof_skl_ops = {
 	.dbg_dump	= hda_dsp_dump,
 
 	/* stream callbacks */
-	.host_stream_open = hda_dsp_pcm_open,
-	.host_stream_close = hda_dsp_pcm_close,
-	.host_stream_hw_params = hda_dsp_pcm_hw_params,
-	.host_stream_trigger = hda_dsp_pcm_trigger,
+	.pcm_open	= hda_dsp_pcm_open,
+	.pcm_close	= hda_dsp_pcm_close,
+	.pcm_hw_params	= hda_dsp_pcm_hw_params,
+	.pcm_trigger	= hda_dsp_pcm_trigger,
 
 	/* firmware loading */
 	.load_firmware = hda_dsp_cl_load_fw,
-- 
2.7.4

